spec/ParseRelation.spec.js

```diff
@@ -517,7 +517,7 @@ describe('Parse.Relation testing', () => {
 
         // Parent object is un-fetched, so this will call /1/classes/Car instead
         // of /1/classes/Wheel and pass { "redirectClassNameForKey":"wheels" }.
-        return query.find(origWheel.id);
+        return query.find();
```

Comment:
Interesting change - what is this?

Reply:
Something’s weird, I checked the Parse SDK and there’s no reference supporting find(objectId). The call signature doesn’t even mention it. It seems that using find with an objectId used to work before, but now it throws an error. The test was using it incorrectly.